aspnetcore/migration/22-to-30.md

```diff
@@ -473,6 +471,8 @@ To use Json.NET in an ASP.NET Core 3.0 project:
                 new CamelCasePropertyNamesContractResolver());
   ```
 
+  **Note:** If you call `AddNewtonsoftJson` without installing [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson), the method fails without throwing an exception. The method also fails silently if you install the wrong Newtonsoft package. Install [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson), not [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/).
```

Comment:
Stating **not** to install `Newtonsoft.Json` makes it sound like: if you have it installed, remove it. Folks very well might want to use `Newtonsoft.Json` within their apps and can add that package if they wish.

They just can't **only** have that package and need the `https://nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson` adapter package.

> The method also fails silently if you install the wrong Newtonsoft package.

Is that what the failure state looks like? @pranavkm, @BrennanConroy

Reply:
That sounds wrong, how is the method available without the package installed? It should be a compile time error, at least it is for SignalR.

Reply:
I think this doc update is in response to https://github.com/dotnet/aspnetcore/issues/13564#issuecomment-534627765

I have not verified this myself though, thought one of you would know the behavior already :)

Reply:
That sounds like mismatches in preview versions. Do you continue to see reports of this (we haven't in AspNetCore)? I wouldn't document something that doesn't apply to the LTS release.